transform/mattermost-analytics/models/intermediate/product/trial_requests/int_cloud_trial_requests_history.sql

```diff
@@ -28,6 +28,7 @@ subscriptions as (
 )
 select
     'stripe:' || subscriptions.subscription_id as trial_request_id
+    , null as server_id
```

Comment:
Perhaps we can get server_id for some of the cases by combining installation id and telemetry.

Reply:
there is also `_int_server_installation_id_bridge` - for the servers marked as cloud from telemetry but with no matching stripe data.